src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -32,7 +32,9 @@
         [rn/view {:style {:flex 1}}
          [account-switcher/view
           {:type     :wallet-networks
-           :on-press #(rf/dispatch [:wallet/close-account-page])}]
+           :on-press (fn []
+                       (rf/dispatch [:wallet/clean-send-data])
```

Comment:
why we would do this here? 🤔

Reply:
should this cleanup not go in the close-account-page event?

Reply:
To remove the remaining data. In the flow:

Account page -> Long press token to bridge -> Bridge -> Back.

The token selected to send is still in the DB.

We can't remove the token when going back from the `Bridge <TOKEN> to` screen because it will break the flow when we start from the bridge button in the CTAs.

Reply:
Thanks 🙏

Reply:
> should this cleanup not go in the close-account-page event?

Yes, I can dispatch it inside too, I'll update it, thanks for spotting it :+1:

Reply:
@ulisesmac `(rf/dispatch [:wallet/clean-send-data])` needs to be inside the `unmount`, and the x button on-press should just be `navigate-back`. Please see this PR https://github.com/status-im/status-mobile/pull/20183/

Reply:
Done :+1: